hat/assets/js/apps/Iaso/domains/completenessStats/CompletenessStatsFilters.tsx

```diff
@@ -156,10 +154,22 @@ export const CompletenessStatsFilters: FunctionComponent<Props> = ({
         [handleChange],
     );
 
+    const periodTypePlaceHolder = useMemo(() => {
+        const messages = {
+            [PERIOD_TYPE_PLACEHOLDER]: MESSAGES.periodPlaceHolder,
+            [NO_PERIOD]: MESSAGES.noPeriodPlaceHolder,
+        };
+
+        return messages[periodType]
+            ? formatMessage(messages[periodType])
+            : undefined;
+    }, [formatMessage, periodType]);
+
```

Comment:
I don't understand why you're using `useMemo` in such a simple computation?

What's the benefit?

For what I know:

- the re-rendering mechanism of React is often fast enough by default
- `useMemo` is best used in case of cost expensive computations which could lead to a re-rendering delay (500ms and more)

Reply:
You are right, there is no need to use useMemo here.